SingularityUI/app/components/dashboard/RequestCount.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, PropTypes } from 'react';
+
+class RequestCount extends Component {
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+    count: PropTypes.number.isRequired,
+    link: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+    this.displayName = 'RequestCount';
```

Comment:
You can do this as a pure function: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md